File: Cine/gestiorAplicacion/tiendaAbst/Tienda.java
package Cine.gestiorAplicacion.tiendaAbst;

import java.util.HashMap;
import java.util.Map;

public abstract class Tienda {

    protected String nombre;
    protected Map<String, Integer> inventario = new HashMap<String, Integer >();
    protected Map<String, Integer> productos = new HashMap<String, Integer >();

    public Tienda(String nombre) {
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public Map<String, Integer> getInventario() {
        return inventario;
    }

    public Map<String, Integer> getProductos() {
        return productos;
    }

    public void setProductos(Map<String, Integer> productos) {
        this.productos = productos;
    }

    public void setInventario(Map<String, Integer> inventario) {
        this.inventario = inventario;
    }

    



}

File: Python/gestionAplicacion/tiendapy/tienda.py
class tienda:
    def __init__(self, nombre, inventario, productos):
        self.nombre = nombre
        self.inventario = inventario
        self.productos = productos

    def getNombre(self):
        return self.nombre

    def getInventario(self):
        return self.inventario

    def getProductos(self):
        return self.productos

    def setNombre(self, nombre):
        self.nombre = nombre

    def setInventario(self, inventario):
        self.inventario = inventario

    def setProductos(self, productos):
        self.productos = productos

    def __str__(self):
        return self.nombre